src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+       class="logo"
+       href="#home">
+        <img
+         src="images/logo.png"
+         alt="logo"
+         class="logo">
+      </a>
```

Comment:
```suggestion
      <a
         class="logo"
         href="#home"
       >
       
        <img
           src="images/logo.png"
           alt="logo"
           class="logo"
         >
      </a>
```
fix code style all over the page